### PR TITLE
Few improvement for SonarQube Scanners

### PR DESCRIPTION
**Pull request by henryju**
* The name of the 2 tools was changed.
* The groupId of SonarQube Runner was also changed

Relates to:
 * https://jira.sonarsource.com/browse/SONARJNKNS-229
 * https://jira.sonarsource.com/browse/SONARJNKNS-230

### CODE REVIEW

File: sonarqubescanner.groovy [hunk 1]
```diff
@@ -41,14 +41,34 @@ def listFromNewUrl() {
...
             url = p.getFullyQualifiedUrl(a.hrefAttribute + "sonar-runner-dist-" + ver + ".zip");
-            return ["id":ver, "name": "SonarQube Runner " + ver, "url":url.toExternalForm()]
+            return ["id":ver, "name": "SonarQube Scanner " + ver, "url":url.toExternalForm()]
+        }
+        return null;
+    }
+}
+
+def listFromNewSonarSourceUrl() {
+    def url = "http://repo1.maven.org/maven2/org/sonarsource/scanner/cli/sonar-scanner-cli/";
```
Comment (daniel-beck) [inline sonarqubescanner.groovy]: Should ideally use HTTPS.

File: sonarqubescanner.groovy [hunk 2]
```diff
@@ -21,13 +21,13 @@ def listFromOldURL() {
...
-            return ["id":ver, "name": "Sonar Runner " + ver, "url":url.toExternalForm()]
+            return ["id":ver, "name": "SonarQube Scanner " + ver, "url":url.toExternalForm()]
         }
         return null;
     }
 }
 
-def listFromNewUrl() {
+def listFromNewCodehausUrl() {
     def url = "http://repo1.maven.org/maven2/org/codehaus/sonar/runner/sonar-runner-dist/";
```
Comment (daniel-beck) [inline sonarqubescanner.groovy]: Also HTTPS.

File: sonarqubescannermsbuild.groovy [hunk 3]
```diff
@@ -2,7 +2,7 @@
 // Generates server-side metadata for MSBuild SonarQube Runner
 import net.sf.json.*
 
-def url = "https://api.github.com/repos/SonarSource/sonar-msbuild-runner/releases".toURL()
+def url = "https://api.github.com/repos/SonarSource-VisualStudio/sonar-msbuild-runner/releases".toURL()
```
Comment (daniel-beck) [inline sonarqubescannermsbuild.groovy]: I wonder whether we'll hit the API limits any time soon.

**Comment by daniel-beck** [comment]
LGTM. Ideally you'd take the opportunity to HTTPS-ify the script. Otherwise, this looks like it can be merged.

**Comment by henryju** [comment]
Updated to use HTTPS. Thanks for feedback. Regarding GitHub API limit I have no idea :)

**Comment by aheritier** [comment]
LGTM too

**Comment by henryju** [comment]
Just a reminder: I don't have commit rights so I can't do the merge myself ;)

**Comment by henryju** [comment]
@daniel-beck @aheritier Can one of you merge the P/R please?

**Comment by henryju** [comment]
@orrc Thanks!

**Pull request merged**